src/main/resources/application.properties

```diff
@@ -3,7 +3,7 @@ spring.jpa.database=POSTGRESQL
 spring.datasource.platform=postgres
 spring.datasource.url=jdbc:postgresql://localhost:5432/CovidTracer
 spring.datasource.username=postgres
-spring.datasource.password=Serenity99!
+spring.datasource.password=Jaeakecc7
```

Comment:
Good to know you guys like to have different passwords for postgres (devs just do postgres for both the username and password most of the time). 
You should probably remove this password value, then rename the file to application.example.properties, and just .gitignore application.properties file

Basically you never want to check in anyone's passwords or the application's passwords.